### ISSUE TITLE
Reproduced the CartoonGan

### ISSUE THREAD

**Issue by longxiong2016** [Yijunmaverick/CartoonGAN-Test-Pytorch-Torch#4]
Hello！brother! I have reproduced the CartoonGan, according to the content of the paper, but the result is not good, some content of the original image have distortion, and the the training is a little shaky, do you have some idea?

**Comment by Yijunmaverick** [comment]
Not try that yet. You may ask the authors. See if you can start the training with the pretrained models first.

**Comment by MenghanXia** [comment]
Yes, I met the same problem. In my opinion, it is because the CartoonGAN is only feasible to those photos that have a very similar appearance to cartoon style themselves. One point you can check is that all the real-world photos illustrated in their paper have such a characteristics. More briefly, the reason why it works in their paper is because they built a special dataset (both for training and testing).  After all, there is little contribution in their network structure and training scheme.

**Comment by FlyingGoblin** [comment]
@longxiong2016 maybe you can send me some of your result?  so that I can give you some advice 0-0
p.s. I am sorry I didn't clean up my code because of the graduation. I will try my best to upload my code in this month.

**Comment by FlyingGoblin** [comment]
@MenghanXia I am sorry : (
